.scalafmt.conf

```diff
@@ -0,0 +1,5 @@
+version = 2.1.1
+align = more    // For pretty alignment.
+maxColumn = 120
+rewrite.rules = [SortImports]
```

Comment:
How do the scalafix and scalafmt rules on sorting imports interact with one another? Do we need both?

Reply:
Oh that's a good question -- it seems from fuzzy recollections of things I've seen tweeted that TypeLevel people are using the superpowered import sorter that Kathryn found, which is 100% through ScalaFix. That one requires some configuration to keep it from writing one import per line (exploding `{Foo, Bar, Baz}` imports) but we could get by with just that one

Reply:
Cool - it seemed like the scalafix rule had more options and features. It might make sense to keep them both if the scalafix rule gets them mostly correct so if folks have format on save enabled in their editors then it'll help.

Reply:
I wound up removing the scalafmt rule because I didn't want to have the configuration in two places and possibly have cases where they disagree failing (eventual) CI. Also, scalafix + metals is coming: https://contributors.scala-lang.org/t/scalafix-future/4247 :soon:

Reply:
a91142d